full-stack-tests/core/src/backend/backend.ts

```diff
@@ -81,6 +81,7 @@ async function init() {
 
   const iModelHost = new IModelHostConfiguration();
   iModelHost.hubAccess = new IModelHubBackend(CloudEnv.cloudEnv.imodelClient);
+  iModelHost.cacheDir = path.join(__dirname, ".cache");  // Set local cache dir
```

Comment:
Are the contents of this dir being purged somewhere after all the tests complete? I couldn't find it.

Reply:
I guess it  suffices that `npm run clean` will delete the cache dir.